templates/common/_base/units/kubelet-auto-node-size.service.yaml

```diff
@@ -5,6 +5,7 @@ contents: |
   Description=Dynamically sets the system reserved for the kubelet
   Wants=network-online.target
   After=network-online.target ignition-firstboot-complete.service
+  After=rhcos-growpart.service # https://issues.redhat.com/browse/OCPBUGS-15087
```

Comment:
Thanks yeah, but unfortunately none of this will work today without fixing our use of `network-online.target`.

I don't see a small targeted fix we can do here.